RubberduckTests/Symbols/SelectedDeclarationProviderTests.cs

```diff
@@ -527,7 +527,7 @@ public void Identify_NamedParameter_Parameter_FromExcel()
                 .AddProjectToVbeBuilder()
                 .Build();
 
-            var (expected, actual) = DeclarationsFromParse(vbe.Object, DeclarationType.Parameter, "Link", "EXCEL.EXE;Excel._Worksheet.Paste");
+            var (expected, actual) = DeclarationsFromParse(vbe.Object, DeclarationType.Parameter, "Link", "EXCEL.EXE;Excel.Worksheet.Paste");
```

Comment:
Do we really need to change this? Seems unrelated to the PR and it's failing in AppVeyor.

Reply:
I changed it because it was the only test out of the several thousand (and I am thrilled to see so many tests!) that failed.  I investigated the test, and I felt it was a typo or mistake.  I'm accustomed to making sure all active tests pass on my PRs, so I changed it.  I didn't mean anything else by it.  :-)